### PR TITLE
feat: add mappings and utils for mtpelerin ramp

### PR DESCRIPTION
**Pull request by jmurphy5613**

### CODE REVIEW

**Comment by gomesalexandre** [comment]
@jmurphy5613 please use conventional commits for your PR title/commits: https://www.conventionalcommits.org/en/v1.0.0/
I've renamed the title of this one as an example.

File: packages/caip/src/adapters/mtpelerin/index.ts [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    btcAssetId,
+    ethAssetId,
+  } from '../../constants'
+
+
+const AssetIdToMtPelerinTickerMap = {
+    [avalancheAssetId]: 'AVAX',
+    [btcAssetId]: 'BTC',
+    [ethAssetId]: 'ETH',
+} as Record<AssetId, string>
```
Comment (gomesalexandre) [inline packages/caip/src/adapters/mtpelerin/index.ts:14]: Can we update the PR description with a reference to the MtPelerin docs where these are defined?

**Review by gomesalexandre — COMMENTED** [review]
Comment

File: packages/caip/src/adapters/mtpelerin/index.ts [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+const invert = <T extends Record<string, string>>(data: T) =>
+  Object.entries(data).reduce((acc, [k, v]) => ((acc[v] = k), acc), {} as Record<string, string>)
+
+const MtPelerinTickerToAssetIdMap = invert(AssetIdToMtPelerinTickerMap)
+
+export const MtPelerinTickerToAssetId = (ticker: string) => MtPelerinTickerToAssetIdMap[ticker]
+
+export const AssetIdToMtPelerinTicker = (assetId: string) => AssetIdToMtPelerinTickerMap[assetId]
```
Comment (gomesalexandre) [inline packages/caip/src/adapters/mtpelerin/index.ts:23]:
```suggestion
export const assetIdToMtPelerinTicker = (assetId: string) => AssetIdToMtPelerinTickerMap[assetId]
```

File: packages/caip/src/adapters/mtpelerin/index.ts [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+    [btcAssetId]: 'BTC',
+    [ethAssetId]: 'ETH',
+} as Record<AssetId, string>
+
+const invert = <T extends Record<string, string>>(data: T) =>
+  Object.entries(data).reduce((acc, [k, v]) => ((acc[v] = k), acc), {} as Record<string, string>)
+
+const MtPelerinTickerToAssetIdMap = invert(AssetIdToMtPelerinTickerMap)
+
+export const MtPelerinTickerToAssetId = (ticker: string) => MtPelerinTickerToAssetIdMap[ticker]
```
Comment (gomesalexandre) [inline packages/caip/src/adapters/mtpelerin/index.ts:21]:
```suggestion
export const mtPelerinTickerToAssetId = (ticker: string) => MtPelerinTickerToAssetIdMap[ticker]
```

**Review by gomesalexandre — CHANGES_REQUESTED** [review]

**Comment by 0xdef1cafe** [comment]
closing in favor of https://github.com/shapeshift/lib/pull/986

**Pull request closed without merging**